docs/reference/search/search-your-data/semantic-search-elser.asciidoc

```diff
@@ -49,31 +49,78 @@ to index the ELSER output.
 
 NOTE: ELSER output must be ingested into a field with the `rank_features` field 
 type. Otherwise, {es} interprets the token-weight pairs as a massive amount of 
-fields in a document. If you get an error similar to this `"Limit of total fields [1000] has been exceeded while adding
-new fields"` then the ELSER output field is not mapped properly and it has a 
-field type different than `rank_features`.
+fields in a document. If you get an error similar to this 
+`"Limit of total fields [1000] has been exceeded while adding new fields"` then 
+the ELSER output field is not mapped properly and it has a field type different 
+than `rank_features`.
 
 [source,console]
 ----
 PUT my-index
 {
   "mappings": {
     "properties": {
-      "ml.tokens": {
-        "type": "rank_features" <1>
+      "ml.tokens": { <1>
+        "type": "rank_features" <2>
       },
-      "text": { <2>
-        "type": "text" <3>
+      "text": { <3>
+        "type": "text" <4>
       }
     }
   }
 }
 ----
 // TEST[skip:TBD]
-<1> The field that contains the prediction is a `rank_features` field.
-<2> The name of the field from which to create the sparse vector representation. 
+<1> The name of the field to contain the generated tokens.
+<2> The field to contain the tokens is a `rank_features` field.
+<3> The name of the field from which to create the sparse vector representation. 
 In this example, the name of the field is `text`.
-<3> The field type which is text in this example.
+<4> The field type which is text in this example.
+
+
+[discrete]
+[[save-space]]
```

Comment:
Can this section be moved to the bottom of the file? I feel like it is a digression from the flow of the tutorial and is something to consider once the user has got started with ELSER